packages/dashboards/app/styles/navi-dashboards/components/navi-dashboard.less

```diff
@@ -46,10 +46,13 @@
   }
 
   .dashboard-actions {
+    button,
     .btn {
+      cursor: pointer;
       font-family: @font-family-sans-serif;
+      margin-right: 15px;
       a {
```

Comment:
\n

Reply:
I wonder if this should go in our common styles, instead of having to do this everywhere?

Reply:
I think the color for `a` elements varies across navi so it's probably best to just handle it on a component level